apps/maestro/src/server/routers/erc20/getERC20TokenBalanceForOwner.ts

```diff
@@ -1,8 +1,9 @@
+import { getFullnodeUrl } from "@mysten/sui.js/client";
+import { SuiClient } from "@mysten/sui/client";
```

Comment:
Can you merge these imports? Please import from `@mysten/sui` instead of `@mysten/sui.js`. I have removed the `@mysten/sui.js` dependency (deprecated one) in [this pr](https://github.com/axelarnetwork/axelarjs/pull/477).